bob/src/cluster/quorum.rs

```diff
@@ -77,6 +78,35 @@ impl Quorum {
         }
     }
 
+    async fn delete_on_nodes(&self, key: BobKey, without_aliens: bool) -> Result<(), Error> {
+        debug!("DELETE[{}] ~~~DELETE LOCAL NODE FIRST~~~", key);
+        let res = delete_at_local_node(&self.backend, key, without_aliens).await;
```

Comment:
When `without_aliens` is true, we should check, that the current node contains target vdisk for key. Otherwise the error will be returned (https://github.com/qoollo/bob/blob/master/bob-backend/src/pearl/core.rs#L295).
But probably we should remove  `without_aliens` flag at all.